data_prep/introspector.py

```diff
@@ -298,7 +372,7 @@ def get_project_funcs(project_name: str) -> Dict[str, List[Dict]]:
         fuzz_target_funcs[fuzz_target_file] = []
       if _contains_function(fuzz_target_funcs[fuzz_target_file], target_func):
         continue
-      _postprocess_function(target_func)
+      target_func = _postprocess_function(target_func, project_name)
```

Comment:
I presume we need remove `target_func = ` because `_postprocess_function` does not return any value now?

Reply:
Yes ,thanks :)